docs/operations/export_api.md

```diff
@@ -455,80 +504,22 @@ status:
 ```
 Note in this example we are in the `example` namespace in the source cluster, which is why the internal links domain ends with `.example.svc`. The external links are what will be visible to outside of the source cluster, so we can use that for when we import into the target cluster.
 
-Now we are ready to import this disk into the target cluster. In order for CDI to import, we will need to provide it the CA certificate that signed the Ingress/Route that we will be connecting to. Luckily the `cert` field of the external links will contain the entire certificate chain. You can use the contents of the `cert` field to populate the configMap. In the target cluster create the configmap in the namespace you will import the disk into:
+Now we are ready to import this disk into the target cluster. In order for CDI to import, we will need to provide appropriate yaml that contains the following:
+- CA cert (as config map)
+- The token needed to access the disk images in a CDI compatible format
+- The VM yaml
+- DataVolume yaml (optional if not part of the VM definition)
 
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: router-cert
-data:
-  ca.pem: |
-    -----BEGIN CERTIFICATE-----
-    ...
-    -----END CERTIFICATE-----
-```
+virtctl provides an additional argument to the download command called `--manifest` that will retrieve the appropriate information from the export server, and either save it to a file with the `--output` argument or write to standard out. By default this output will not contain the header secret as it contains the token in plaintext. To get the header secret you specify the `--include-secret` argument. The default output format is `yaml` but it is possible to get `json` output as well.
 
-Next create a secret in the same namespace on the target cluster. Note: the token value will be a header that is passed to the server by CDI.
+Assuming there is a running Virtual Machine export called `example-export` and the same namespace exists in the target cluster. The name of the kubeconfig of the target cluster is named `kubeconfig-target`, to clone the vm into the target cluster run the following commands:
 
-```yaml
-apiVersion: v1
-kind: Secret
-metadata:
-  name: secret-headers
-stringData:
-  token: "x-kubevirt-export-token:1234567890ab"
+```bash
+$ virtctl vmexport download example-export --manifest --include-secret --output=import.yaml
+$ kubectl apply -f import.yaml --kubeconfig=kubeconfig-target
```

Comment:
> --kubeconfig=kubeconfig-target

If the kubeconfig was exported, it should work without this, right?

Reply:
No, we are apply the generated manifest to the target cluster which has a different kubeconfig than the source cluster which we used for the first command.

Reply:
Okay, it's needed if we are accessing the target cluster from the same tab